.github/workflows/doc-build.yml

```diff
@@ -82,7 +87,38 @@ jobs:
                 set -o pipefail;
                 make html | tee log.txt;
         -   name: Upload documentation
-            uses: actions/upload-artifact@v3
+            uses: actions/upload-artifact@v4
             with:
-                name: doc
+                name: DocHTML
```

Comment:
Why this change ?

Reply:
I think it's better to be explicit about the format as sphinx can compile to various forms + it's consistent with other repos in Parietal-INRIA / mind-inria.